src/microbe_stage/editor/upgrades/ChemoreceptorUpgradeGUI.tscn

```diff
@@ -1,91 +1,108 @@
-[gd_scene load_steps=4 format=2]
+[gd_scene load_steps=5 format=2]
 
 [ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=1]
 [ext_resource path="res://src/gui_common/TweakedColourPicker.tscn" type="PackedScene" id=2]
 [ext_resource path="res://src/microbe_stage/editor/upgrades/ChemoreceptorUpgradeGUI.cs" type="Script" id=3]
+[ext_resource path="res://src/gui_common/fonts/Lato-Regular-Small.tres" type="DynamicFont" id=4]
 
-[node name="ChemoreceptorUpgradeGUI" type="VBoxContainer"]
-anchor_right = 1.0
-anchor_bottom = 1.0
+[node name="ChemoreceptorUpgradeGUI" type="ScrollContainer"]
```

Comment:
I think the changes in this file should now be reverted as the scroll container is in the parent GUI again.